src/common/pure/CurrencyInputPanel/CurrencyInputPanel.tsx

```diff
@@ -154,12 +153,7 @@ export function CurrencyInputPanel(props: CurrencyInputPanelProps) {
           </div>
           <div>
             <styledEl.FiatAmountText>
-              <FiatValue
-                isLoading={isRateLoading}
-                priceImpactLoading={priceImpactLoading}
-                fiatValue={fiatAmount}
-                priceImpact={priceImpact}
-              />
+              <FiatValue priceImpactParams={priceImpactParams} fiatValue={isRateLoading ? null : fiatAmount} />
```

Comment:
There was a single usage of `isLoading` prop. Essentially, `isLoading` is the same with `!!fiatAmount`